docs/quick_start/installation.rst

```diff
@@ -63,18 +63,18 @@ default, cmake will try to install all of the items labelled with * and so
 it is not necessary to install those items manually:
 
 - cmake >= 3.12
-- \*Expat >= 2.2.5 (XML parser for CDL/CLF/CTF)
+- \*Expat >= 2.2.8 (XML parser for CDL/CLF/CTF)
 - \*yaml-cpp >= 0.6.3 (YAML parser for Configs)
-- \*IlmBase (Half only) >= 2.3.0 (for half domain LUTs)
+- \*IlmBase (Half only) >= 2.4.0 (for half domain LUTs)
 - \*pystring >= 1.1.3
 
 Some optional components also depend on:
 
 - \*Little CMS >= 2.2 (for ociobakelut ICC profile baking)
-- \*pybind11 >= 2.4.3 (for the Python bindings)
+- \*pybind11 >= 2.6.1 (for the Python bindings)
 - Python >= 2.7 (for the Python bindings)
 - Python 3.7 or 3.8 (for the docs, with the following PyPi packages)
-    - Sphinx
```

Comment:
FWIW: The casing of these packages is consistent with PyPI: https://pypi.org/project/Sphinx/

That being said, I don't object to consistent casing here since `pip` is case insensitive.

Reply:
Good point, I reverted the change!